ftl/core/deck-config.ftl

```diff
@@ -202,6 +206,9 @@ deck-config-show-answer-timer-tooltip =
 
 deck-config-audio-title = Audio
 deck-config-disable-autoplay = Don't play audio automatically
+deck-config-disable-autoplay-tooltip =
+    If you check this setting, Anki will not play audio
+    until you press the replay audio key, r or F5.
```

Comment:
```suggestion
    automatically. It can be played manually by clicking/tapping on an audio icon, or by using the replay audio action.
```

(we need to avoid desktop-specific terminology)